packages/edit-site/src/components/sync-state-with-url/use-init-edited-entity-from-url.js

```diff
@@ -62,6 +87,22 @@ function useResolveEditedEntityAndContext( { postId, postType } ) {
 				postTypeToResolve,
 				postIdToResolve
 			) {
+				// For the front page, we alwayse use the front page template if existing.
```

Comment:
```suggestion
				// For the front page, we always use the front page template if existing.
```

I can help it. It's a sickness.

Reply:
I can't help but make mistakes :P